content/en/blog/2024/kubecon-na.md

```diff
@@ -0,0 +1,119 @@
+  2024 3:25pm - 4:00pm MST
+- **[Distributed Tracing with Jaeger and OpenTelemetry](https://sched.co/1hoyb)**<br>
+  by Jonah Kowall, Independent & Pavol Loffay, Red Hat<br> Wednesday November
+  13, 2024 4:30pm - 5:05pm MST
+- **[Understanding How OpenTelemetry Network Uses eBPF for Network Observability](https://sched.co/1how7)**<br>
+  by Shivanshu Raj Shrivastava, SigNoz & Jonathan Perry, PerfPod<br> Wednesday
+  November 13, 2024 5:25pm - 6:00pm MST
+- **[The OTTL Cookbook: A Collection of Solutions to Common Problems](https://sched.co/1i7mZ)**<br>
+  by Tyler Helmuth, Honeycomb & Evan Bradley, Dynatrace<br> Wednesday November
+  13, 2024 5:25pm - 6:00pm MST
+- **[Lesson’s Learned Adopting OpenTelemetry at Scale](https://sched.co/1i7nC)**<br>
+  by Alex Arnell, Heroku / Salesforce<br> Thursday November 14, 2024 11:00am -
+  11:35am MST
+- **[How We Made OpenTelemetry Be Our Fitness Tracker for Your CI/CD Pipelines!](https://sched.co/1i7nM)**<br>
+  by Nicolas Woerner, Clario & Andreas Grabner, Dynatrace<br> Thursday November
+  14, 2024 11:00am - 11:35am MST
+- **[Low-Overhead, Zero-Instrumentation, Continuous Profiling for OpenTelemetry](https://sched.co/1i7nz)**<br>
+  by Christos Kalkanis, Elastic<br> Thursday November 14, 2024 2:30pm - 3:05pm
+  MST
+- **[Fluent Bit: Better Pipelines for Observability](https://sched.co/1hoxB)**<br>
+  by Eduardo Silva, Chronosphere<br> Thursday November 14, 2024 4:30pm - 5:05pm
+  MST
+- **[Mastering OpenTelemetry Collector Configuration](https://sched.co/1i7ot)**<br>
+  by Steve Flanders, Cisco<br> Thursday November 14, 2024 4:30pm - 5:05pm MST
+- **[Tutorial: OpenTelemetry Hands-on - Automatic and Manual Instrumentation for Java and Python Apps](https://sched.co/1i7pd)**<br>
+  by Matthias Haeussler, Novatec Consulting GmbH & Tiffany Jernigan,
+  Independent<br> Friday November 15, 2024 11:00am - 12:30pm MST
+- **[Contribfest: OpenTelemetry Contribfest](https://sched.co/1hoyF)**<br> by
+  OpenTelemetry Maintainers<br> Friday November 15, 2024 4:00pm - 5:30pm MST
+
+## Observability Day
+
+_[Observability Day][] fosters collaboration, discussion, and knowledge sharing of
+cloud-native observability projects_. This event will be held on November 12, 2024
+from 9am to 6pm. [Check the full schedule](https://colocatedeventsna2024.sched.com/overview/type/Observability+Day)
+to find your favorite talks about Observability and OpenTelemetry.
+
+> <i class="far fa-exclamation-triangle"></i> **IMPORTANT access note**: You
+> need an _in-person all-access_ pass for on-site access to **Observability
+> Day**. For details, see [KubeCon registration][]. If you have a virtual ticket,
+> you will be able to follow **Observability Day** through a live stream.
+
+## OpenTelemetry Observatory
+
+Drop by and say _"Hi!"_ at OpenTelemetry Observatory presented by Splunk in the
+Expo Hall. This will be a place for informal chats, meetups, and other
+discussions led by OpenTelemetry community members and maintainers.
+
```

Comment:
The schedule is coming together! Can you please add the following info about these 2 new session types we're hosting:

> New for this year! 
> * **SIG Office Hours**: Come meet the people behind the project, learn about their work, ask questions, share feedback, or find out how to contribute. 
> * **Special Topic Discussions**: Come learn about two of the _hottest_ OTEPs right now, Events and Entities!